extensions/cloudfiles-extensions/src/test/java/io/druid/storage/cloudfiles/CloudFilesObjectApiProxyTest.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Druid - a distributed column store.
+ *  Copyright 2012 - 2015 Metamarkets Group Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.druid.storage.cloudfiles;
+
+import org.easymock.EasyMockSupport;
+import org.jclouds.io.Payload;
+import org.jclouds.openstack.swift.v1.domain.SwiftObject;
+import org.jclouds.openstack.swift.v1.features.ObjectApi;
+import org.jclouds.rackspace.cloudfiles.v1.CloudFilesApi;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import static org.easymock.EasyMock.*;
+
+public class CloudFilesObjectApiProxyTest extends EasyMockSupport {
+
+	@Test
+	public void getTest() {
+		final String path = "path";
+		final String region = "region";
+		final String container = "container";
+
+		CloudFilesApi cloudFilesApi = createMock(CloudFilesApi.class);
+		ObjectApi objectApi = createMock(ObjectApi.class);
+		SwiftObject swiftObject = createMock(SwiftObject.class);
+		Payload payload = createMock(Payload.class);
```

Comment:
I've just noticed that in this test the payload is never opened. The opening and the closing of the payload is tested in `CloudFilesByteSourceTest`.